packages/hierarchy-builder/src/hierarchy-builder/HierarchyNode.ts

```diff
@@ -311,6 +283,47 @@ export namespace HierarchyNodeKey {
   }
 }
 
+/** @beta */
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export namespace HierarchyNodeKey {
```

Comment:
please move this right under the `HierarchyNodeKey` type

Reply:
Moved.